man/insertUI.Rd

```diff
@@ -0,0 +1,95 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/insert-ui.R
+\name{insertUI}
+\alias{insertUI}
+\title{Insert UI objects}
+\usage{
+insertUI(selector, multiple = FALSE, where = c("beforeBegin", "afterBegin",
+  "beforeEnd", "afterEnd"), ui, immediate = FALSE, container = if (inline)
+  "span" else "div", inline = FALSE, session = getDefaultReactiveDomain())
+}
+\arguments{
+\item{selector}{A string that is accepted by jQuery's selector (i.e. the
+string \code{s} to be placed in a \code{$(s)} jQuery call). This selector
+will determine the element(s) relative to which you want to insert your
+UI object.}
+
+\item{multiple}{In case your selector matches more than one element,
+\code{multiple} determines whether Shiny should insert the UI object
+relative to all matched elements or just relative to the first
+matched element (default).}
+
+\item{where}{Where your UI object should go relative to the selector:
+\describe{
+  \item{\code{beforeBegin}}{Before the selector element itself}
+  \item{\code{afterBegin}}{Just inside the selector element, before its
+  first child.}
+  \item{\code{beforeEnd}}{Just inside the selector element, after its
+  last child}
+  \item{\code{afterEnd}}{After the selector element itself (default)}
+}
+Adapted from
+\href{https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentHTML}{here}.}
+
+\item{ui}{The UI object you want to insert. This can be anything that
+you usually put inside your apps's  \code{ui} function.}
+
+\item{immediate}{Whether the UI object should be immediately inserted into
+the app when you call \code{insertUI}, or whether Shiny should wait for the
+next time your app is updated for some other reason (default).}
```

Comment:
That does sound better. I wasn't sure how to say "flush cycle" in layman's terms, but I think this is pretty good. Thanks! Changed it.